media-gfx/sane_backends/sane_backends-1.3.1.recipe

```diff
@@ -78,20 +78,20 @@ BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:automake
 	cmd:autoreconf
-	cmd:autopoint
+	cmd:autopoint$secondaryArchSuffix
 	cmd:awk
 	cmd:find
 	cmd:gcc$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
-	cmd:python3
+	cmd:python$pythonVersion
 	"
 
 BUILD()
 {
 	if [ $effectiveTargetArchitecture == 'x86' ]; then
-		export CFLAGS="-DTIFF_DISABLE_DEPRECATED"
+		export CFLAGS="-DTIFF_DISABLE_DEPRECATED -O2"
 	fi
```

Comment:
The default CFLAGS are `-O2 -g` for autotools, (see https://github.com/haikuports/haikuporter/commit/b251aa1b42cc3291d1b299a9fe1e81d0c415874e) so only setting them for x86 causes that to be built without debuginfo, and with debuginfo for all others. I would suggest to make it consistent for all architectures.

```suggestion
	if [ $effectiveTargetArchitecture == 'x86' ]; then
		export CFLAGS="-DTIFF_DISABLE_DEPRECATED -O2"
	else
		export CFLAGS="-O2"
	fi
```

If `-g` is added, a debuginfo package should also be defined.

Reply:
ow bugger, missed the arch check, could you push the change to fix this? that is probably why it didn't fail on 64bit the first run.  (kinda busy atm) :)

Reply:
done (untested)

Reply:
Thanks!